### ISSUE TITLE
ImportError: No module named PIL

### ISSUE THREAD

**Issue by roberttaylor426** [powlo/weather.metoffice#2]
Hi,

I've recently tried to install your plugin onto my fresh installation of RaspBMC. When I try it tries to update the weather forecast I get a script error reported, and I see the following in the logs: 

09:39:31 T:2900001856   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.ImportError'>
                                            Error Contents: No module named PIL
                                            Traceback (most recent call last):
                                              File "/home/pi/.kodi/addons/weather.metoffice/src/metoffice/default.py", line 21, in <module>
                                                import utilities, properties, urlcache
                                              File "/home/pi/.kodi/addons/weather.metoffice/src/metoffice/properties.py", line 6, in <module>
                                                from PIL import Image
                                            ImportError: No module named PIL
                                            -->End of Python script error report<--

Any idea?

**Comment by powlo** [comment]
I don't think I made PIL a dependency because xbmc's python seemed to have it already built in.  Or maybe I had another script that had already installed it.

Maybe put in a <requires><import addon="script.module.pil" /></requires> in the addon.xml

**Comment by roberttaylor426** [comment]
Sorry, could you please clarify your last sentence? :-)

**Comment by powlo** [comment]
Hmm, my xml got chomped out.

What I meant to say was , maybe put a dependency on script.module.pil in the addon.xml

```xml
<requires>
	<import addon="script.module.pil">
  </requires>
```

Looking at the [repo](https://github.com/xbmc/xbmc/tree/master/addons), script.module.pil seems to be built in, so I'm not sure if declaring a dependency helps.

I saw Martijns comment [here](http://www.reddit.com/r/xbmc/comments/2pnob2/python_imaging_library_pil_on_kodi_on_android/cmzgcqb):

>There is a general issue on android for getting PIL installed. There's currently a fix pending which might hit for 14.1

**Comment by MartijnKaijser** [comment]
PIL is fixed for upcoming 14.2 and indeed you need to import it in addon.xml if you want to use it
